omod/pom.xml

```diff
@@ -12,6 +12,26 @@
     <name>Open Concept Lab Module OMOD</name>
     <description>OMOD project for OpenConceptLab</description>
 
+    <profiles>
+        <profile>
+            <id>2.2</id>
+            <dependencies>
+                <dependency>
+                    <groupId>org.openmrs.module</groupId>
+                    <artifactId>legacyui-omod</artifactId>
+                    <version>${legacyuiVersion}</version>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
+                    <groupId>javax.servlet</groupId>
+                    <artifactId>javax.servlet-api</artifactId>
+                    <version>4.0.1</version>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+        </profile>
+    </profiles>
+
```

Comment:
If / when we merge this, we should add a stage to Bamboo to build and test the 2.2 profile

Reply:
We should do the same for Github Actions, so we can catch it on PRs to...